include/roboteam_ai/stp/new_plays/AttackingPass.h

```diff
@@ -60,12 +61,18 @@ class AttackingPass : public Play {
      * Calculates the pass location
      * @return the pass location
      */
-    const Vector2 calculatePassLocation() noexcept;
+    std::pair<Vector2, double> calculatePassLocation(Grid searchGrid) noexcept;
```

Comment:
document the proper return type please, what's the Vector2 in this case, what's the double?

You could make a named struct for it, like

```cpp
struct pass_location_t {
    Vector2 location;
    double speed;
};```

Reply:
I don't agree with making a named struct for it. Although I definitely agree that it should be better documented.